components/services/bat_ads/bat_ads_client_mojo_bridge.cc

```diff
@@ -253,6 +253,40 @@ void BatAdsClientMojoBridge::RunDBTransaction(
       base::BindOnce(&OnRunDBTransaction, std::move(callback)));
 }
 
+void BatAdsClientMojoBridge::ClearScheduledCaptcha() {
+  if (!connected()) {
+    return;
+  }
+
+  bat_ads_client_->ClearScheduledCaptcha();
+}
+
+void OnGetScheduledCaptcha(ads::GetScheduledCaptchaCallback callback,
+                           const std::string& captcha_id) {
+  std::move(callback).Run(captcha_id);
+}
+
+void BatAdsClientMojoBridge::GetScheduledCaptcha(
+    const std::string& payment_id,
+    ads::GetScheduledCaptchaCallback callback) {
+  if (!connected()) {
+    return std::move(callback).Run("");
+  }
+
+  bat_ads_client_->GetScheduledCaptcha(
+      payment_id, base::BindOnce(&OnGetScheduledCaptcha, std::move(callback)));
```

Comment:
there's no reason for this when everything is `base::Bind`. As above you can just pass the callback directly

Reply:
Good catch, fixed.